clusters/build-clusters/02_cluster/openshift-monitoring/cluster-monitoring-config_configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
```

Comment:
Really, this is how its adjusted?

Reply:
yes. for 4.4.
Is it adjustable? I have never tested it.
I know there are some immutable fields.
Worst case, delete and recreate.